### DISCUSSION TITLE
Why 'Everyone should avoid dairy'?

### DISCUSSION THREAD

**Discussion by davbl** [nikivdev/go#62] [General]
Referring to: https://github.com/nikitavoloboev/knowledge/blob/master/health/nutrition/nutrition.md

1. Do you think that because of the chemical composition of milk, or because of how dairy products are currently made (in your country)?
2. Or just, why do you think that?
3. Plus please add source(s) of that information, if possible.

Thank you Nikita.

**Comment by nikivdev** [comment]
I agree that it shouldn't say `everyone` as blanket statement, I changed it to just `avoid dairy`.

As far as my reasoning, it's just things I read about it and the fact that cows are involved in its production (vegan bias).

I'd love to hear sources proving otherwise and I will try find reputable sources to argue my case against not consuming dairy for health/ethical reasons. 

Thanks for bringing this up.
